SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -203,7 +211,9 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
             TwilioVoice.handleNotification(payload.dictionaryPayload, delegate: self)
         }
         
-        completion()
+        DispatchQueue.main.async {
+            completion()
```

Comment:
We should not use iOS 11 APIs when `IPHONEOS_DEPLOYMENT_TARGET`is set to 10.0.

https://developer.apple.com/documentation/pushkit/pkpushregistrydelegate/2875784-pushregistry